README.md

```diff
@@ -196,6 +199,7 @@ function CommonRegistry(opts){
 util.inherits(CommonRegistry, DefaultRegistry);
 
 CommonRegistry.prototype.init = function(takerInst){
+  DefaultRegistry.prototype.init.call(this, takerInst);
```

Comment:
I did notice super.init doesn't do anything -- now... Figured this might allow more flexibility going forward, knowing that changing default registry's internals won't break user stuff (at least those following the pattern from this doc.)